sdk/openai/azure-ai-openai/src/samples/java/com/azure/ai/openai/StreamingChatSample.java

```diff
@@ -3,10 +3,7 @@
 
 package com.azure.ai.openai;
 
-import com.azure.ai.openai.models.ChatCompletions;
-import com.azure.ai.openai.models.ChatCompletionsOptions;
-import com.azure.ai.openai.models.ChatMessage;
-import com.azure.ai.openai.models.ChatRole;
+import com.azure.ai.openai.models.*;
```

Comment:
<img width="727" alt="image" src="https://github.com/Azure/azure-sdk-for-java/assets/45607042/810b7a23-dc8f-4f9e-9420-52fb7d24e793">

Reply:
Thank you, Shawn! I somehow always forget about this setting!